apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -23,11 +23,8 @@ import {
   Maybe,
   Pool,
   Scalars,
-  useGetMePoolsQuery,
-  RoleAssignment,
   LocalizedString,
   useAllPoolsQuery,
-  Team,
```

Comment:
I think the team column is broken now.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a2b0811d-a911-4628-84d8-9de30573950d)

Reply:
Oooo, good catch! 🙇‍♀️  Fixed in [query for team display name on pools table](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7449/commits/9e4fa79521cf734a8ac44e5d8b484fbf4660b95c)

Reply:
Looks great!